lib/markdown-stream.js

```diff
@@ -276,19 +272,7 @@ module.exports = {
             IO.txt.val = editor.getText();
           });
         });
-
-        editorView
-          .off('click')
-          .on('click', function()
-          {
-            IO.log('editorView:click ');
-            IO.txt.val = editor.getText();
-
-          });
```

Comment:
I could not determine the point of the above events so I removed them since `on` is now deprecated. If needed back perhaps [onDidChangeCursorPosition](https://atom.io/docs/api/v0.210.0/TextEditor#instance-onDidChangeCursorPosition) would do as an appropriate replacement.